source/ddbus/meson.build

```diff
@@ -0,0 +1,35 @@
+lib_src = [
+    'attributes.d',
+    'bus.d',
+    'c_lib.d',
+    'conv.d',
+    'exception.d',
+    'package.d',
+    'router.d',
+    'simple.d',
+    'thin.d',
+    'util.d',
+]
+
+ddbus_lib = library(
+    'ddbus',
+    lib_src,
+    install: true,
+    version: meson.project_version(),
```

Comment:
Well, we don't need to overwrite it here too. `version` gets inherited from the project version by default.

Reply:
I don't think so, if version isn't set for the library only libddbus.so is installed without symlinks